camayoc/tests/qpc/cli/utils.py

```diff
@@ -197,14 +210,16 @@ def source_add_and_check(options, inputs=None, exitstatus=0):
         options['cred'] = ' '.join(options['cred'])
     if 'hosts' in options:
         options['hosts'] = ' '.join(options['hosts'])
+    if 'exclude-hosts' in options:
+        options['exclude-hosts'] = ' '.join(options['exclude-hosts'])
     if 'type' not in options:
         options['type'] = 'network'
     command = 'qpc source add'
     for key, value in options.items():
         if value is None:
             command += ' --{}'.format(key)
         else:
-            command += ' --{}={}'.format(key, value)
+            command += ' --{} {}'.format(key, value)
```

Comment:
I am curious why changing from using `=` to ` `

Reply:
So it turns out that the way commands are parsed was breaking if there was more than one parameter passed. So `--hosts 192.168.1.1 192.168.2.2` works, but `--hosts=192.168.1.1 192.168.2.2` eats up the first IP address and then fails to parse the second.